rebar.config

```diff
@@ -1,5 +1,5 @@
 {deps,
- [{cowlib, {git, "https://github.com/ninenines/cowlib", {tag, "2.7.0"}}},
+ [{cowlib, {git, "https://github.com/ninenines/cowlib", {tag, "2.8.0"}}},
```

Comment:
could you put a comment here to document why this app needs to depend on cowlib.